pages/common/comm.md

```diff
@@ -21,4 +21,4 @@
 
 - Print lines only found in second file, when the files aren't sorted:
```

Comment:
This description is no longer accurate when you omit the sorting step in the command.

Reply:
Couldn't I assume the files are sorted?

Reply:
Yes, but then you would need to change the description into 

`Print lines only found in the second file when the files are sorted`.

Reply:
```suggestion
- Print lines only found in the second file, when both files are sorted:
```